### PR TITLE
UI update

### PR DESCRIPTION
**Pull request by ng5jr**
Desktop and mobile design updated for Home, Profile and User screens.

### CODE REVIEW

**Comment by pseudozach** [comment]
@ng5jr here's a list of things we need to resolve:
- donation text gets cut off
- npubkey etc QR codes are not centered
- when there are no relays, `relay` text should also disappear
- can we change NIP5 checkmark with an @ sign instead?
- can you also add badges for telegram and github as we've added these recently. 

if you can do last 2, I can look into the others. Also need to re-do nip7 signing popups we have for new design.
https://github.com/pseudozach/nostr.directory/blob/master/src/templates/Profile.tsx#L1610

**Comment by pseudozach** [comment]
thanks @ng5jr we're almost there I think,

* badge click should open link in new tab
* we can make the back button just go back one with `router.back()`
* can we have small popup/snackbar something when things are copied? just some visual feedback that copy worked for instance for twitter verification popup
* can we use the same twitter verification popup design in the badge popups as well? Just to be consistent.
* If we end up using the same popup everywhere this wont be a problem: Profile helper popup font-sizes are not the same. telegram dialog font seems larger to me.
* the hover color changes on the bottom buttons on homepage are great but they're not happening on the main CTA buttons. is it possible to add?

File: src/templates/Profile.tsx [hunk 1]
```diff
@@ -771,220 +960,180 @@ const Profile = () => {
...
-                      ),
-                      button1: <></>,
-                      button2: 'ok',
-                    })
+
+              {/* Add information for github */}
+              <div className="my-2 flex items-center w-full md:w-max">
+                <BadgeCard
+                  variant={'github'}
+                  verified={!!tweet.telegram}
```
Comment (pseudozach) [inline src/templates/Profile.tsx]: @ng5jr  github badge is checking telegram. this should check for githubIdentity which is a local check and then popup shows telegram but it should show github verification instructions.

**Comment by pseudozach** [comment]
@ng5jr sorry just 1 more thing, can you add a button for search as well, old version had an arrow button there.
and we'll need to match the error alert dialogs to this theme, maybe make the corners round as well.

**Comment by ng5jr** [comment]
Conflicts solved

**Pull request merged**